go.mod

```diff
@@ -30,4 +31,7 @@ require (
 	k8s.io/utils v0.0.0-20200729134348-d5654de09c73
 )
 
-replace vbom.ml/util => github.com/fvbommel/util v0.0.0-20180919145318-efcd4e0f9787
+replace (
+	github.com/openshift/library-go => github.com/hexfusion/library-go v0.0.0-20201110203057-2161de7a0c28
```

Comment:
This is temporary, I assume, to test locally.

Reply:
yes, this is proof PR for https://github.com/openshift/library-go/pull/940 so I have vendored that PR to validate the change.